### PR TITLE
fix: Functions without prototype error

### PR DESCRIPTION
**Pull request by brustolin**
Xcode 14.3 is complaining everywhere about this:

image

_#skip-changelog_

### CODE REVIEW

File: Sources/SentryCrash/Recording/SentryCrashC.c [hunk 1]
```diff
@@ -75,6 +75,7 @@ onCrash(struct SentryCrash_MonitorContext *monitorContext)
 {
     SentryCrashLOG_DEBUG("Updating application state to note crash.");
     sentrycrashstate_notifyAppCrash();
+    printf("### CRASHED");
```
Comment (philipphofmann) [inline Sources/SentryCrash/Recording/SentryCrashC.c:78]: `m`: Maybe you added this by mistake?

**Review by philipphofmann — APPROVED** [review]
LGTM

File: Sources/SentryCrash/Recording/SentryCrashC.c (hunk 1 above)
Comment (brustolin) [inline Sources/SentryCrash/Recording/SentryCrashC.c:78]:
```suggestion
```

**Comment by philipphofmann** [comment]
As this fixes a bug @brustolin, I think we must add a changelog entry.

**Pull request merged**